src/writer.ts

```diff
@@ -55,6 +66,27 @@ export const handleSpaceCreated: CheckpointWriter = async ({
   await mysql.queryAsync(query, [item]);
 };
 
+export const handleMetadataUriUpdated: CheckpointWriter = async ({ rawEvent, event, mysql }) => {
+  if (!event || !rawEvent) return;
+
+  console.log('Handle space metadata uri updated');
+
+  const space = validateAndParseAddress(rawEvent.from_address);
```

Comment:
It's a bit strange to still require `rawEvent`, feel like this should be part of `event`.

Reply:
We probably would need to make `event` a wrapper with parsed property (which we didn't like initially) because it's very likely that otherwise it will conflict with `from_address`  that is actually part of an event.

Reply:
Yes, and `parsed` may conflict with event too, it probably need something like this:
```
event.raw
event.from_address
event.params.proposal_id // Similar than subgraph
```